Core/GDCore/IDE/Project/ProjectResourcesCopier.cpp

```diff
@@ -26,20 +26,20 @@ bool ProjectResourcesCopier::CopyAllResourcesTo(
     bool preserveAbsoluteFilenames,
     bool preserveDirectoryStructure) {
 
+  // Check if there are some resources with absolute filenames
+  gd::ResourcesAbsolutePathChecker absolutePathChecker(originalProject.GetResourcesManager(), fs);
+  gd::ResourceExposer::ExposeWholeProjectResources(originalProject, absolutePathChecker);

```

Comment:
I think this is useless since quite a bit of time (it was used years ago to ask the user what to do with absolute filenames, but pretty sure it's not used anymore)